SmartEVSE2.X/EVSE.c

```diff
@@ -242,7 +242,7 @@ unsigned char OldButtonState = 0x0f;
 unsigned char LCDNav = 0;
 unsigned char SubMenu = 0;
 unsigned long ScrollTimer = 0;
-unsigned char LCDpos = 0;
+char LCDpos = 0;
```

Comment:
Why unsigned char to signed char?

Reply:
Went back to Master version and removed all "far" qualifiers.
Could not reproduce any issue with LCDpos being unsigned char, so indeed no need to change!
Therefore also no need to change in EVSE.h
Please disregard!